python/example_code/secretsmanager/secretsmanager_basics.py

```diff
@@ -101,401 +38,34 @@ def get_value(self, stage=None):
                  contained in the `SecretString` field. When the secret is bytes,
                  it is contained in the `SecretBinary` field.
         """
-        if self.name is None:
-            raise ValueError
-
         try:
             kwargs = {"SecretId": self.name}
             if stage is not None:
                 kwargs["VersionStage"] = stage
-            response = self.secretsmanager_client.get_secret_value(**kwargs)
+            self.secretsmanager_client.batch_get_secret_value(
```

Comment:
None of the arguments look correct.

For example,

> [SecretIdList](https://docs.aws.amazon.com/secretsmanager/latest/apireference/API_BatchGetSecretValue.html#API_BatchGetSecretValue_RequestSyntax)
> The ARN or names of the secrets to retrieve. You must include Filters or SecretIdList, but not both.

It should not be "string", as I understand it

Reply:
You're totally right and this file will actually be deleted by [this PR](https://github.com/awsdocs/aws-doc-sdk-examples/pull/5809). I removed the changes from the PR completely.